### DISCUSSION TITLE
Should nVidia show Optimus when GHelper is in Standard mode?

### DISCUSSION THREAD

**Discussion by slikvik55** [seerge/g-helper#2553] [Q&A] [answered]
I've just noticed that in Standard made, nVidia shows this:
![image](https://github.com/seerge/g-helper/assets/17570255/266a9542-2ba8-4c02-b8a2-ef2c321a9617)

You homepage says Optimus, so I'm wondering what's correct. I've never touched this setting in nVidia.

**Comment by seerge** [comment, marked as answer]
@slikvik55 hello,

**This is exactly correct.** 

Standard mode is when your dGPU is enabled - and it's totally up to Nvidia Drivers and Optimus how to use it / and if your internal display should be driven by iGPU or dGPu :)

When you set Ultimate, it would use Asus own MUX switch, to route display directly to dGPU all the time.

Thanks.
